src/dbnode/storage/shard_fetch_blocks_metadata_test.go

```diff
@@ -297,3 +299,68 @@ func TestShardFetchBlocksMetadataV2WithSeriesCachePolicyNotCacheAll(t *testing.T
 		}
 	}
 }
+
+func TestShardFetchBlocksMetadata(t *testing.T) {
+	opts := DefaultTestOptions().SetSeriesCachePolicy(series.CacheRecentlyRead)
+	ctx := opts.ContextPool().Get()
+	defer ctx.Close()
+
+	numOfActiveSeries := 1000
+	writeBatchSize := 100
```

Comment:
same comment about batch sizes not being simple divisors. 

actually. because this is a unit test - can you run sub-tests with multiple different configurations

Reply:
Updated the unit test to run multiple configurations sub tests